src/createServer.js

```diff
@@ -1,3 +1,71 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+/* eslint-disable max-len */
+/* eslint-disable no-console */
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const errorHanding = {
+      errors: [
```

Comment:
It's necessary to do this like that? I mean errors array inside a object. Just const error = []; will be not enough?

Reply:
From Readme file 
```
#### Validation
Text in the URL and query param `toCase` are mandatory. Also, `toCase` value should be one of the supported cases.
If something is not correct, you should respond with 400 status, `Bad request` statusText and the next payload:
```json
{
  "errors": [
    {
      "message": "<SPECIFIC MESSAGE TEXT HERE>"
    }
  ],
}
```
Array of messages can contain more than one error. For example, if both text and case are not provided.
```

This why I decided to do it in this way